### PR TITLE
Parallelize crawling of the eventful api for speed.

### PR DESCRIPTION
**Pull request by lcary**
This PR imports grequests for parallelized crawling of the Eventful API.

Additionally, I've updated requirements.txt and the README while making this change.

### CODE REVIEW

**Comment by nickspeal** [comment]
Travis looks like it is failing because the Spotify token for nickspeal_test has changed. I'll fix this later.

**Comment by lcary** [comment]
@nickspeal Okay, thanks. This should be ready to merge now.

File: music_this_week_app/tests/test_eventFinder.py [hunk 1]
```diff
@@ -14,8 +15,52 @@
...
+        self.assertEqual(request_was_successful(ResponseStub(ok=False, _json={})), False)
+
+    def test_parse_events(self):
+        event_dict = {
+            'url': 'http://foo.com',
+            'description': 'bar',
+            'id': 0,
+            'title': 'foo',
+            'performers': {'performer': [{'name': 'Bono'}]},
+            'venue_name': 'the foo bar',
```
Comment (nickspeal) [inline music_this_week_app/tests/test_eventFinder.py:50]: 😂

**Comment by nickspeal** [comment]
Seems legit. I'll test it on my own repo and then PR from there.

**Pull request closed without merging**